Cargo.toml

```diff
@@ -51,7 +51,8 @@ clap_mangen = { version = "0.1", optional = true }
 cpio = "0.2.1"
 flate2 = "^1.0"
 glob = "^0.3"
-gptman = { version = ">= 0.7, < 0.9", default-features = false }
+# disable default-enabled cli in gptman 0.x
+gptman = { version = ">= 0.7, < 2", default-features = false }
```

Comment:
It looks like this syntax might only find versions less than 2. Do we want to include version 2?

Reply:
No, that's intentional.  Assuming [semver](https://semver.org/) is respected by the upstream project, there shouldn't be any breaking changes in 1.x releases, but 2.0 will include breaking changes.  Those may or may not affect our particular use of the library, but we shouldn't claim compatibility with 2.0 until it exists and we've had a chance to test it.